app/scripts/lib/notification-manager.js

```diff
@@ -29,17 +29,20 @@ class NotificationManager {
       // bring focus to existing chrome popup
       await this.platform.focusWindow(popup.id)
     } else {
-      const { screenX, screenY, outerWidth, outerHeight } = window
-      const notificationTop = Math.round(screenY + (outerHeight / 2) - (NOTIFICATION_HEIGHT / 2))
-      const notificationLeft = Math.round(screenX + (outerWidth / 2) - (NOTIFICATION_WIDTH / 2))
+      const lastFocused = await this.platform.getLastFocusedWindow()
```

Comment:
Could you add a fallback for when this fails? The `getLastFocused` API isn't available on Firefox for Android, which... does technically work with the extension right now (although not very well, and not for very much longer). But still... might as well not break it?

A `try / catch` block with some fallback position should do the trick.

Reply:
Any advice on how to test this locally for someone who bleeds apple? (Aka I own 0 android devices ;p) @Gudahtt Would like to make sure that the fallback actually works. I suppose I could like modify the global object in the console before triggering the popup.

Reply:
Yep, that sounds like it'd work! You could also try throwing an error in `getLastFocusedWIndow` directly, as I would expect that function to throw an exception on Firefox for Android (since `window.getLastFocused` would be undefined)

Reply:
Added the try/catch, and did `delete chrome.windows.getLastFocused` to confirm the behavior. This will open the popup in the top left corner on devices that don't support it -- unless the originating window in android chrome is not a background process like it is on desktop chrome. In which case it should position it in the top right corner.